torch/_inductor/fx_passes/freezing_patterns.py

```diff
@@ -114,17 +115,17 @@ def register_binary_folding_pattern(pattern, extra_check=_return_true):
 
 @functools.lru_cache(None)
 def addmm_patterns_init():
-    if torch.cuda.is_available():
-        # workaround https://github.com/pytorch/pytorch/issues/97894
-        device = "cuda"
-    else:
-        device = "cpu"
-        if not config.cpp.enable_concat_linear:
-            return
-
+    # workaround https://github.com/pytorch/pytorch/issues/97894
```

Comment:
this workaround comment is not relevant..

Reply:
resolved, thanks.